docs/source/conf.py

```diff
@@ -44,7 +44,7 @@
 
 # General information about the project.
 project = "cognite-sdk"
-copyright = "2018, Erlend Vollset"
+copyright = "2019, Cognite"
```

Comment:
According to https://cognitedata.atlassian.net/wiki/spaces/COG/pages/7143447/Open+Source+Policy it should be "Cognite AS"

Reply:
Fixed